packages/react/src/components/DataTable/tools/normalize.js

```diff
@@ -55,7 +55,7 @@ const normalize = (rows, headers, prevState = {}) => {
         isEditing: false,
         isValid: true,
         errors: null,
-        hasSlugHeader: !!slug,
+        hasSlugHeader: !!(slug || decorator?.type?.displayName === 'AILabel'),
```

Comment:
wondering if we should deprecate `hasSlugHeader` prop in TableCell in favor for `hasAILabelHeader`

Reply:
This is for internal use only. But yes, we can definitely change the name. Thanks for the suggestion